help/change-your-timezone.md

```diff
@@ -16,4 +20,7 @@ You can change the time zone Zulip displays to others.
 
 2. Under **Profile**, select a time zone.
 
+To stop receiving prompts when your computer’s time zone differs from the
+device’s, check the box for '**Disable prompts when the computer’s time zone is different**'.
```

Comment:
I dropped the /help/ commit, since this isn't our standard style for doing /help/ documentation. @laurynmm can you put up a PR for it?